pkg/provision/providers/qemu/launch.go

```diff
@@ -439,13 +439,19 @@ func launchVM(config *LaunchConfig) error {
 			return err
 		}
 
+		tpmDevice := "tpm-tis,tpmdev=tpm0"
+
+		if Arch(config.Architecture) == ArchArm64 {
```

Comment:
nit: we can make this a method on `Arch`: 

```go
func (a Arch) TMPDeviceArg() string
```

Reply:
I restructured more code to be defined on `Arch`, so we pass around less stuff